app/src/main/res/values/strings.xml

```diff
@@ -272,6 +272,7 @@
 
     <string name="pref_highlight_entry_title">Highlight tokens when tapped</string>
     <string name="pref_highlight_entry_summary">Make tokens easier to distinguish from each other by temporarily highlighting them when tapped</string>
+    <string name="pref_minimize_on_copy_title">Minimize app after copying token</string>
```

Comment:
Let's make the title a bit shorter, and the summary a bit longer. For the title: "Minimize on copy".